src/xcode/ENA/ENA/Source/Scenes/Checkin/CheckInDetail/CheckinDetailViewController.xib

```diff
@@ -0,0 +1,375 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this view needed?
If yes, could you give it a name?

<img width="508" alt="Bildschirmfoto 2021-03-19 um 11 15 30" src="https://user-images.githubusercontent.com/1941000/111765297-7e4e3200-88a4-11eb-816c-a85b8a5842ac.png">

Reply:
since this view lies on top of two different color views, the color ENAHairlineContrast will show the background view like this image, so i hade to add an extra view to fix this 
![Screenshot 2021-03-19 at 11 24 37](https://user-images.githubusercontent.com/15270737/111766325-b43fe600-88a5-11eb-9923-a58e8868acce.png)

Reply:
how it looks without the view
![Screenshot 2021-03-19 at 11 29 41](https://user-images.githubusercontent.com/15270737/111770812-2e269e00-88ab-11eb-9060-7b5fe9a490c6.png)